src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/RawlsBillingProjectComponentSpec.scala

```diff
@@ -5,190 +5,93 @@ import org.broadinstitute.dsde.rawls.model._
 class RawlsBillingProjectComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers {
   import driver.api._
 
-  "RawlsBillingProjectComponent" should "save, load and delete" in withEmptyTestDatabase {
-    val subjId = RawlsUserSubjectId("17, the most random number")
```

Comment:
these tests just use the users in the test fixture now. made this a lot easier to deal with because rawlsBillingProjectQuery.create can only be used if the backing rawls groups already exist in the db- so those are created in the test data now